docs/issue_template.md

```diff
@@ -0,0 +1,11 @@
+### Operating system
+
+### `nbgrader --version`
+
+### Notebook - Help, About version info
+
```

Comment:
Please can you include

    ### `jupyterhub --version` (if used with JupyterHub)